content/en/index.md

```diff
@@ -136,7 +136,7 @@ function Counter() {
 
 <section class="home-split">
     <div>
-        <h2>Todo List Component</h2>
+        <h2>Todo List</h2>
```

Comment:
`Fetch GitHub Stars` is not `Fetch GitHub Stars Component`.
Therefore, This  is consistent.